kubernetes/deployment.tmpl

```diff
@@ -3,14 +3,13 @@ kind: PersistentVolumeClaim
 apiVersion: v1
 metadata:
   name: hamlet-staging-export-tmp
-  annotations:
-    volume.beta.kubernetes.io/storage-class: "aws-efs"
 spec:
   accessModes:
     - ReadWriteMany
+  storageClassName: azurefile
   resources:
     requests:
-      storage: 1Mi
+      storage: 1Gi
```

Comment:
You mentioned 5GB on the phone, is 1 gig enough?

Reply:
Good catch – I did actually intend to set this to 5 GB but 1 GB should be plenty for staging anyway so I think I'll leave it as-is.